clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -259,3 +259,26 @@ export async function executeCommandMultipleTimes(
     await vscode.commands.executeCommand(command);
   }
 }
+
+interface PgmConf {
```

Comment:
Its processor groups specific logic, let's have it in processorgroups.spec.test.ts